ports/esp32/usb.c

```diff
@@ -28,7 +28,7 @@
 #include "py/mphal.h"
 #include "usb.h"
 
-#if CONFIG_USB_OTG_SUPPORTED && !CONFIG_ESP_CONSOLE_USB_CDC && !CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG_ENABLED
+#if MICROPY_HW_USB_CDC
```

Comment:
I like how much simpler this has become.  But again it will clash with #15108, and maybe should be `MICROPY_HW_ENABLE_USBDEV`?  (That's not what #15108 does, but might be what #15108 needs.)